src/Components/Components/src/RenderTree/ArrayBuilderSegment.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
Isn't it on for the whole project? Does this make some other difference that I'm missing?

Reply:
This type and a few others are shared with Ignitor which hasn't been updated as yet. We should be able to remove this once everything is sorted.